src/common/base/Status.h

```diff
@@ -98,10 +98,7 @@ class Status final {
     STATUS_GENERATOR(SyntaxError);
 
     // TODO(dangleptr) we could use ErrorOr to replace SpaceNotFound here.
-    STATUS_GENERATOR(SpaceNotFound);
```

Comment:
Why?

Reply:
We have remove them from `meta.thrift`'s `ErrorCode` and replace with `E_NOT_FOUND`

Reply:
Why remove them?  Any benefits?